src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final String CRYPTO_ALGORITHM = "SHA-512";
+
+    public static byte[] getSalt() {
+        SecureRandom secureRandom = new SecureRandom();
+        byte[] salt = new byte[16];
```

Comment:
magic number

Reply:
This is a common literal that I use once in code.

Reply:
Yes, you use it once, but i think your code would be more readable if you make this field constant

Reply:
I don't agree with you. A person who knows basic java will 100% understand that an array of 16 elements is being created here. I think a constant only makes sense if it is reused in the code.

Reply:
done.